$NetBSD: patch-ab,v 1.2 2009/09/13 15:19:02 yacht Exp $

Indicate to the preprocessor magic further down in this file that NetBSD also
has a working alloca.

--- src/common/3dsparse/aseFile.tab.cpp.orig	2009-02-23 03:14:20.000000000 +0000
+++ src/common/3dsparse/aseFile.tab.cpp
@@ -5,7 +5,8 @@
 
 #define YYBISON 1  /* Identify Bison output.  */
 
-#if !defined(__FreeBSD__) && !defined(__DARWIN__) && !defined(_WIN32)
+#if !defined(__FreeBSD__) && !defined(__DARWIN__) && !defined(_WIN32) && \
+    !defined(__NetBSD__)
 #define alloca 
 #endif //!defined(__FreeBSD__) && !defined(__DARWIN__)
 
